index.html

```diff
@@ -0,0 +1,16 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+
+<h1>Hello world!</h1>
+
+
+</body>
```

Comment:
Try not to forget about good code formatting, even on simple projects:
```suggestion
<body>
  <h1>Hello world!</h1>
</body>
```